docs/appendix/changelog.md

```diff
@@ -307,7 +307,7 @@ The updates to the camera component have improved the process of connecting to a
 The latest updates enable you to:
 
 - View readable webcam names in the **video path** of your camera component.
-- Specify your preferred framerate by selecting the desired value in the newly added **framerate field** on the **Config** tab.
+- Specify your preferred framerate by selecting the desired value in the newly added **framerate field** on the **CONFIGURE** tab.
```

Comment:
Did you ask whether it'll stay all caps already?

Reply:
Now have & yes